Marlin/src/gcode/host/M115.cpp

```diff
@@ -245,6 +245,18 @@ void GcodeSuite::M115() {
     // CONFIG_EXPORT
     cap_line(F("CONFIG_EXPORT"), ENABLED(CONFIGURATION_EMBEDDING));
 
+    // FSR_PROBING
+    cap_line(F("FSR_PROBING"), ENABLED(HAS_FSR) || ENABLED(HAS_Z_MIN_ADC));
```

Comment:
`HAS_FSR` & `HAS_Z_MIN_ADC` do not exist in Marlin.

Reply:
Removed for this PR.

I will add support for FSR / HAS_Z_MIN_ADC under a separate PR at some point.